### ISSUE TITLE
Instructions to update the docker-compose container

### ISSUE THREAD

**Issue by xeruf** [ampache/ampache-docker#84]
I think these should be added to the readme:

<https://stackoverflow.com/questions/49316462/how-to-update-existing-images-with-docker-compose/49316987#49316987>  

    sudo docker-compose pull
    sudo docker-compose up --force-recreate --build -d

I run these in a cronjob to keep ampache up to date.
An option to disable the update notification in the interface would be handy there, as the docker image is often updated a few days later

**Comment by divStar** [comment]
While I totally agree with you, I imagine a pull request would be handy 😄 . Thanks for the info though, because that's just what I was looking for.

**Comment by xeruf** [comment]
I think I need to refine these, though - my ampache now has a broken database connection, I'm not sure why...

**Comment by SuperFola** [comment]
Hi @xeruf, have you figured why your database connection was broken? I'm also interested in upgrading the container but I don't want to break stuff

**Comment by xeruf** [comment]
No, I just waited and it worked again :sweat_smile: the cronjob runs every night and I did not have any issues anymore in the recent weeks.
